<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLDB</class>
 <widget class="QDialog" name="DialogLDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxData">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxGenotype">
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genotype</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGenotype"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonGenotype">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBlock">
        <property name="text">
         <string>Block:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditBlock"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonBlock">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMAF">
        <property name="text">
         <string>Min. haplotype frequency:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditMAF">
        <property name="text">
         <string>0.01</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMaxDist">
        <property name="text">
         <string>Max. block length:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMaxDist">
        <property name="text">
         <string>200000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLowCI">
        <property name="text">
         <string>Low CI for strong LD:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLowCI">
        <property name="text">
         <string>0.7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelHighCI">
        <property name="text">
         <string>High CI for strong LD:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditHighCI">
        <property name="text">
         <string>0.98</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelRecHighCI">
        <property name="text">
         <string>High CI for recombination:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditRecHighCI">
        <property name="text">
         <string>0.9</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelInfoFrac">
        <property name="text">
         <string>Min. Informative strong LD:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditInfoFrac">
        <property name="text">
         <string>0.95</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxGenotype</tabstop>
  <tabstop>lineEditGenotype</tabstop>
  <tabstop>pushButtonGenotype</tabstop>
  <tabstop>lineEditBlock</tabstop>
  <tabstop>pushButtonBlock</tabstop>
  <tabstop>lineEditMAF</tabstop>
  <tabstop>lineEditMaxDist</tabstop>
  <tabstop>lineEditLowCI</tabstop>
  <tabstop>lineEditHighCI</tabstop>
  <tabstop>lineEditRecHighCI</tabstop>
  <tabstop>lineEditInfoFrac</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLDB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
